server/controllers/finance/reports/collection_capacity/index.js

```diff
@@ -36,6 +36,26 @@ async function report(req, res, next) {
     const INVOICE_TRANSACTION_TYPE = 11;
     const INCOME_ACCOUNT_TYPE = 4;
 
+    const formatedDateFrom = moment(dateFrom).format('YYYY-MM-DD');
+    const formatedDateTo = moment(dateTo).format('YYYY-MM-DD');
+
+    const includeUnpostedValues = qs.includeUnpostedValues ? Number(qs.includeUnpostedValues) : 0;
+    let generalTable = 'general_ledger';
+
+    if (includeUnpostedValues) {
+      generalTable = `
+      (
+        SELECT trans_id, description, trans_date, debit_equiv, credit_equiv, currency_id, debit, credit,
+          account_id, record_uuid, reference_uuid, created_at, transaction_type_id
+        FROM posting_journal WHERE DATE(trans_date) BETWEEN DATE("${formatedDateFrom}") AND DATE("${formatedDateTo}")
+        UNION ALL 
+        SELECT trans_id, description, trans_date, debit_equiv, credit_equiv, currency_id, debit, credit,
+          account_id, record_uuid, reference_uuid, created_at, transaction_type_id
+        FROM general_ledger WHERE DATE(trans_date) BETWEEN DATE("${formatedDateFrom}") AND DATE("${formatedDateTo}")
```

Comment:
I don't think it makes any performance difference, but do you really need all these columns?  For example, you do not use description, currency_id, debit, credit, etc.  It would probably improve readability if you omitted them.